# -*- coding: utf-8 -*-
# Generated by Django 1.11.2 on 2017-07-11 11:01
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('type_page', '0001_initial'),
    ]

    operations = [
        migrations.CreateModel(
            name='FootballType',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('first_team', models.CharField(max_length=64, null=True)),
                ('second_team', models.CharField(max_length=64, null=True)),
                ('draw', models.BooleanField(default=True)),
                ('date_game', models.DateTimeField()),
                ('league', models.CharField(max_length=64)),
                ('course', models.IntegerField()),
                ('bet', models.IntegerField()),
            ],
            options={
                'ordering': ['date_game'],
            },
        ),
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.ASCIIUsernameValidator()], verbose_name='username'),
        ),
    ]
